cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -296,19 +294,15 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             }
         }
 
-        if (evaluatorRun.violations.isNotEmpty()) {
-            println("Rule violations found.")
-            throw ProgramResult(2)
-        }
-    }
-
-    private fun printSummary(errors: List<RuleViolation>) {
-        val counts = errors.groupingBy { it.severity }.eachCount()
+        val counts = evaluatorRun.violations.groupingBy { it.severity }.eachCount()
 
         val errorCount = counts[Severity.ERROR] ?: 0
         val warningCount = counts[Severity.WARNING] ?: 0
         val hintCount = counts[Severity.HINT] ?: 0
 
-        println("Found $errorCount errors, $warningCount warnings, $hintCount hints.")
+        println("Violations found: $errorCount errors, $warningCount warnings, $hintCount hints.")
```

Comment:
I believe we should keep the exact wording of the original `println()`, but move it *inside* the `if`, and after the `if` have have another

    println("Found $hintCount hints only.")